metasrc/tex/lsstdoc.py

```diff
@@ -185,6 +190,41 @@ def is_draft(self):
             return False
 
     @property
+    def revision_datetime(self):
+        """Current revision date of the document (`datetime.datetime`).
+
+        The `revision_datetime_source` describes how the revision date
+        is computed.
+        """
+        if not hasattr(self, '_datetime'):
+            self._parse_revision_date()
+
+        return self._datetime
+
+    @property
+    def revision_datetime_source(self):
+        r"""Data source for the `revision_datetime` attribute (`str`).
+
+        Possible string values are:
+
+        - ``'now'``: The current date, when ``\today`` is the content of
```

Comment:
My reading of the code suggests that this isn't how `now` works. I might be wrong but the code seems to check for an explicit date in the file (checking that it's not `\today`), then looks at `git` and only then falls back to the current day. Do you ever expect `now` to trigger?

Reply:
Oh I totally botched the ``'now'`` bullet point; the second sentence belongs in ``tex``. Should be much clearer now, thanks.